python/heterocl/tools.py

```diff
@@ -84,7 +84,8 @@ def __init__(self):
         self.xpfm = None
         self.binary = None
         self.build_dir = None
-        
+        self.suported_modes.append("csyn")
```

Comment:
Can you also fix the typo (supported_modes) in this file?

Reply:
Sorry, I missed that. Will do soon.